haskell.nix/application/.buildkite/pipeline.yml

```diff
@@ -20,6 +20,14 @@ steps:
   commands:
   - nix build -L .#checks.x86_64-linux.trailing-whitespace
 
+# - label: check-stack2cabal
+#   commands:
+#   - nix develop -c bash -c 'stack2cabal && git add -A && git diff HEAD --exit-code'
```

Comment:
README suggests using `nix develop .#ci`, but here the default devShell is used, I assume that `.#ci` is missing

Reply:
Oh, I also have two additional concerns about `git` usage:
1) This invocation implies that `git` is already available in the `PATH`, which might not be the case, though, this is probably quite unlikely
2) Some CIs may strip `.git` folder and `git diff` might not work, can you check if this will work for supported CIs?